Firmware/cmdqueue.cpp

```diff
@@ -422,25 +424,24 @@ void get_command()
 			  cmdbuffer[bufindw + CMDHDRSIZE] = '$';
 		}
         // if we don't receive 'N' but still see '*'
-        if ((cmdbuffer[bufindw + CMDHDRSIZE] != 'N') && (cmdbuffer[bufindw + CMDHDRSIZE] != '$') && (strchr(cmdbuffer+bufindw+CMDHDRSIZE, '*') != NULL))
+        if ((*cmd_head != 'N') && (cmdbuffer[bufindw + CMDHDRSIZE] != '$') && (strchr(cmd_start, '*') != NULL))
```

Comment:
Why wasn't `cmdbuffer[bufindw + CMDHDRSIZE]` also changed to `*cmd_head`? I don't see a difference yet...

Reply:
Probably because I removed it entirely on #3823. I will fix this